### PR TITLE
Adding Helm Chart for Ingestion

### PR DESCRIPTION
**Pull request by KennethSkylight**
# Helm Chart for Ingestion

Adds the helm chart for ingestion. Uses a load balancer instead of ingress temporarily as we figure out the CI/CD pattern. 

Resolves [ticket #738](https://app.zenhub.com/workspaces/secret-6480bf2ee530095ab41ebbe9/issues/gh/cdcgov/phdi/738)

### CODE REVIEW

File: charts/record-linkage/charts/Chart.yaml [hunk 1]
```diff
@@ -2,15 +2,6 @@ apiVersion: v2
 name: record-linkage-chart
 description: A Helm Chart for the DIBBs Record Linkage Service
 type: application
-
-# This is the chart version. This version number should be incremented each time you make changes
-# to the chart and its templates, including the app version.
-# Versions are expected to follow Semantic Versioning (https://semver.org/)
```
Comment (emmastephenson) [inline charts/record-linkage/charts/Chart.yaml:8]: These changes seem unrelated - is there a reason the comments were removed?

File: ingestion/templates/secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name}}-geocoding-secrets
+type: Opaque
+stringData:
+  auth-id: {{ .Values.smartyAuthId}}
```
Comment (emmastephenson) [inline ingestion/templates/secrets.yaml:7]: With the type of secret being `stringData`, you'll need to change these slightly:
`{{ .Values.smartyAuthId | quote }}`

Otherwise, the parser throws an exception when you try to run `helm install`. 

`* Secret in version "v1" cannot be handled as a Secret: json: cannot unmarshal number into Go struct field Secret.stringData of type string`

File: ingestion/values.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+replicaCount: 1
+
+image:
+  repository: nginx
```
Comment (emmastephenson) [inline ingestion/values.yaml:4]: I think the image repository here is leftover from the generated file that `kubernetes create chart` makes. The repository should probably be our Github container registry, not nginx. (I noticed it was also an issue on the record linkage service, we should change it there too)

**Review by emmastephenson — APPROVED** [review]
Looking good! Just a few things to tweak. Thanks Kenneth!

**Review by emmastephenson — CHANGES_REQUESTED** [review]
Clicked the wrong box 🤦

File: charts/record-linkage/charts/Chart.yaml (hunk 1 above)
Comment (KennethSkylight) [reply charts/record-linkage/charts/Chart.yaml:8]: It was failing the linter. I guess I could just make the comments shorter in lines? I figured we weren't really using it.

File: ingestion/templates/secrets.yaml (hunk 2 above)
Comment (KennethSkylight) [reply ingestion/templates/secrets.yaml:7]: Huh... I didn't get that error when I did the install nor the dry install, I'll take a look

Comment (KennethSkylight) [reply ingestion/templates/secrets.yaml:7]: Changed

**Review by emmastephenson — APPROVED** [review]
Looks good!
